src/app/organizations/collection/collection.component.html

```diff
@@ -101,18 +104,28 @@ <h3 *ngIf="Dockstore.FEATURES.enableNotebooks">Workflows, Notebooks, and Tools</
             <h3 *ngIf="!Dockstore.FEATURES.enableNotebooks">Workflows and Tools</h3>
 
             <div data-cy="collectionEntry" *ngFor="let entry of collection?.entries">
-              <mat-card class="pb-3">
+              <mat-card
```

Comment:
Brainstorming, does the notebook mat card need the `preview` flag?

Reply:
Maybe instead of adding it to the card, I could add the preview bubble to the title since we usually add it on the tab/title, similar to my-starred?

<img src="https://user-images.githubusercontent.com/97123241/233199217-bff85d7e-c39c-4aa8-86f3-5962dd27f0bd.png" width=400px>

![image](https://user-images.githubusercontent.com/97123241/233197587-9185e2a0-0893-43ae-ae35-fc8685914e14.png)

Reply:
I think it's a little ambiguous beside the title because it also mentions workflows and tools. Since you're adding the feature name to the feature warning banner, we can probably live without the preview label on this page

Reply:
Got it, makes sense